src/content/app/species/SpeciesPage.scss

```diff
@@ -11,19 +11,24 @@
   grid-template-columns: 1fr auto;
   align-items: center;
   line-height: 1;
+  padding-left: 11px;
 }
 
 .topbarLeft {
   display: flex;
   align-items: center;
+  cursor: pointer;
+  justify-self: start;
 }
 
-.close {
-  margin-top: 2px;
+.backChevron {
+  margin-left: 5px;
 }
```

Comment:
What distance are you trying to achieve here, and why make it a left margin of the chevron rather than the left padding of one of its parents?

![image](https://user-images.githubusercontent.com/6834224/182556916-f8a6cae5-4ddc-4aa2-bb52-1ab2fbcb4709.png)

Reply:
> What distance are you trying to achieve here, and why make it a left margin of the chevron rather than the left padding of one of its parents?
> 
> ![image](https://user-images.githubusercontent.com/6834224/182556916-f8a6cae5-4ddc-4aa2-bb52-1ab2fbcb4709.png)
> 
> Should the topbar's left padding be adjusted instead? Why is it like it is?
> 
> ![image](https://user-images.githubusercontent.com/6834224/182557142-44593b0b-9e58-4aa5-b98a-7f807f9c1fde.png)

I dont need this margin-left, i forgot to remove it after adding padding to the container. Good spot. I have now removed it.